aws/sqs/process

```diff
@@ -40,7 +40,7 @@ fi
 # Before each build, we lock the message for 20 minutes + 10 seconds
 #   (our maximum allowed build time + a grace period)
 aws sqs change-message-visibility --receipt-handle "$RECEIPT_HANDLE" --queue-url "$QUEUE_URL" --visibility-timeout 1210
- PR="$PR"   ./bench || reportFailure
+FLAMEGRAPH="true" PR="$PR" REF="$HEAD_SHA" ./bench || reportFailure
```

Comment:
This doesn't work, because `$HEAD_SHA` references a commit that does not exist in the base repository 😕